app/models/miq_request.rb

```diff
@@ -14,6 +14,7 @@ class MiqRequest < ApplicationRecord
   belongs_to :requester,         :class_name  => "User"
   belongs_to :tenant
   belongs_to :service_order
+  belongs_to :miq_request
```

Comment:
Since the schema changed, , I assume this should be `belongs_to :parent, :class_name => "MiqRequest"` or something to that effect?  I would expect this to fail the specs as well if we do a cross-repo test.

Reply:
It makes sense to call the `.parent` on a request.